controller/search_spaces_blackbox_test.go

```diff
@@ -85,7 +85,7 @@ func (rest *TestSearchSpacesREST) TestSpacesSearchOK() {
 		{"with special chars", args{offset("0"), limit(10), "&:\n!#%?*"}, expects{totalCount(0)}},
 		{"with * to list all", args{offset("0"), limit(10), "*"}, expects{totalCountAtLeast(len(idents))}},
 		{"with multi page", args{offset("0"), limit(10), "TEST"}, expects{hasLinks("Next")}},
-		{"with last page", args{offset(strconv.Itoa(len(idents) - 1)), limit(10), "TEST"}, expects{hasNoLinks("Next"), hasLinks("Prev")}},
+		{"with last page", args{offset(strconv.Itoa(len(idents) - 1)), limit(100), "TEST"}, expects{hasNoLinks("Next"), hasLinks("Prev")}},
```

Comment:
I found the reason why some spaces were not deleted at the end of the test: the cleaner function was declared in the `SetupSuite()` method instead of the `SetupTest()`. So it would clean only the data from the first test, but not the other ones!